kolibri/plugins/learn/assets/src/views/AssessmentWrapper/index.vue

```diff
@@ -456,15 +456,6 @@ oriented data synchronization.
         updateContentNodeProgress(this.channelId, this.id, this.exerciseProgress);
         this.$emit('updateProgress', this.exerciseProgress);
       },
-      sessionInitialized() {
-        if (this.isUserLoggedIn) {
-          this.callInitMasteryLog();
-        } else {
-          this.createDummyMasteryLog();
-        }
-        this.nextQuestion();
-        this.$emit('sessionInitialized');
-      },
```

Comment:
were these drive-by changes or possibly a mistake? Seems unrelated to the rest of the PR

Reply:
These were intentional - part of the changes I made to the ContentRenderer was to remove the session initialization part, and just make creating the component itself contingent on that happening (as it was almost entirely a logging/tracking concern).